plugins/org.jkiss.dbeaver.ext.mysql.ui/plugin.xml

```diff
@@ -97,7 +97,9 @@
             <supports dataSource="mysql"/>
         </editorContribution>
     </extension>
-
+    <extension point="org.jkiss.dbeaver.sql.executors">
+        <executor class="org.jkiss.dbeaver.ext.mysql.tools.MySQLScriptExecutor" datasource="org.jkiss.dbeaver.ext.mysql.model.MySQLDataSource"/>
```

Comment:
Why is `datasource` specifically require a FQN class name? Every data source has a unique ID. Use it instead.

Reply:
Then you can check whether the given data source extends the specified one.